arangodb/hatch.toml

```diff
@@ -0,0 +1,10 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["38"]
+version = ["3.8"]
+
+[envs.default.overrides]
+matrix.version.env-vars = "ARANGODB_VERSION"
+
+[envs.bench]
```

Comment:
No need if there wasn't already a benchmark env

Reply:
It'll just slow down CI

Reply:
My bad. Updating the current open PRs and taking a look at the one we already merged.